### PR TITLE
Showing overview block text on movie

### PR DESCRIPTION
**Pull request by kristofferjs**

### CODE REVIEW

**Comment by bjoerge** [comment]
Will this display images and other types embedded in the block array?

File: pages/movie.js [hunk 1]
```diff
@@ -32,6 +34,7 @@ export default class Movie extends React.Component {
 
   render() {
     const {movie} = this.props
+    console.log('movie', movie)
```
Comment (bjoerge) [inline pages/movie.js]: This should be removed

**Review by bjoerge — CHANGES_REQUESTED** [review]

**Comment by bjoerge** [comment]
According to the schema, the block array can contain entries of type `image`, `movie`, `screening` and `person`. It would be good if we could illustrate an example of rendering each of these types.

**Comment by kristofferjs** [comment]
Good point. Then we need to populate the movies data with other block types than normal text

**Comment by kristofferjs** [comment]
On hold. until fixing a GROQ issue.

**Comment by kristofferjs** [comment]
Schema is updated without references. Now has plotSummaries and image

**Review by bjoerge — APPROVED** [review]

**Pull request merged**